pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "todo-app"
+version = "0.1.0"
+description = "A simple ToDo app for your DevOps apprenticeship"
+authors = []
+
+[tool.poetry.dependencies]
+python = "^3.7"
+flask = "^1.1.2"
+python-dotenv = "^0.14.0"
+requests = "^2.24.0"
+pytest = "^6.1.2"
```

Comment:
I think selenium is missing from the list of dependencies here

Reply:
Also I wasn't sure if this was still expecting pip to be used, or whether this should now be migrated over to poetry - if the latter then the readme should be updated to reflect that